### DISCUSSION TITLE
Regarding the his is rat tool

### DISCUSSION THREAD

**Discussion by hakimoLYM** [RealBey/ThisIsNotRat#3] [General]
Can you provide me with the password for the tool and why it was closed?
I'm talking about this tool
https://github.com/RealBey/ThisIsNotRat

**Comment by Barikfi** [comment]
> Can you provide me with the password for the tool and why it was closed?
> I'm talking about this tool
> https://github.com/RealBey/ThisIsNotRat

Hi there 
What password ??
I couldn't understand you will you please elaborate.

**Comment by BeyTek** [comment]
> Can you provide me with the password for the tool and why it was closed? I'm talking about this tool https://github.com/RealBey/ThisIsNotRat

"Your_fucking_strong_password"
